components/blitz/src/omero/gateway/facility/BrowseFacility.java

```diff
@@ -84,6 +84,55 @@
      * @return See above.
      * @throws DSOutOfServiceException  If the connection is broken, or logged in.
      */
+    public Collection<DataObject> getHierarchy(SecurityContext ctx, Class rootType,
+            long userId) throws DSOutOfServiceException {
+        ParametersI param = new ParametersI();
+        if (userId >= 0) {
+            param.exp(omero.rtypes.rlong(userId));
+        }
+        param.orphan();
+        return getHierarchy(ctx, rootType, null, param);
+    }
+
+    /**
+     * Retrieves hierarchy trees rooted by a given node.
+     * i.e. the requested node as root and all of its descendants.
+     *
+     * @param ctx The security context.
+     * @param rootType The type of node to handle.
+     * @param rootIDs The node's id.
+     * @param options The retrieval options.
+     * @return See above.
+     * @throws DSOutOfServiceException If the connection is broken, or logged in.
```

Comment:
I see what you mean
That's an error in the doc that was not fixed when migrating methods.
Copy and paste ;-)